integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/K8sEvents.java

```diff
@@ -174,7 +174,9 @@ public static boolean domainEventExists(
     try {
       List<CoreV1Event> events = Kubernetes.listNamespacedEvents(domainNamespace);
       for (CoreV1Event event : events) {
-        if (event.getReason().equals(reason) && (isEqualOrAfter(timestamp, event))) {
+        if (event.getReason().equals(reason) && (isEqualOrAfter(timestamp, event))
+                && (event.getMetadata().getLabels() != null
+                && event.getMetadata().getLabels().containsValue(domainUid))) {
```

Comment:
This will return false for NamespaceWatchingStarted and StopManagingNamespace events. The NamespaceWatchingStarted  event has this labels in metadata
`- metadata:
    creationTimestamp: '2022-03-30T06:03:50Z'
    labels:
      weblogic.createdByOperator: 'true'
    managedFields:`
so we should not be checking for the presence of domainUid in those 2 events

Reply:
fixed